pycbc/events/coinc_rate.py

```diff
@@ -82,10 +82,38 @@ def combination_noise_coinc_rate(rates, slop):
         Value is expected coincidence rate in the combination, units Hz
     """
     # multiply product of trigger rates by the overlap time
-    allowed_area = multiifo_noise_coincident_area(list(rates), slop)
+    log_rates = {k: numpy.log(r) for (k, r) in rates.items()}
+    combo_coinc_rate = combination_noise_coinc_rate_log(log_rates, slop)
+
+    return combo_coinc_rate
+
+
+def combination_noise_coinc_rate_log(log_rates, slop):
+    """
+    Calculate the expected rate of noise coincidences for a combination of
+    detectors
+
+    Parameters
+    ----------
+    log_rates: dict
+        Dictionary keyed on ifo string
+        Value is a sequence of logarithm of single-detector trigger rates,
+        units assumed to be Hz
+    slop: float
+        time added to maximum time-of-flight between detectors to account
+        for timing error
+
+    Returns
+    -------
+    combo_coinc_rate: numpy array
+        Value is expected coincidence rate in the combination, units Hz
+    """
+    # multiply product of trigger rates by the overlap time
+    allowed_area = numpy.log(multiifo_noise_coincident_area(list(log_rates),
```

Comment:
would prefer to calculate allowed area as before then take the log in the line that calculates combo_coinc_rate (and maybe call that lograte rather than rate).

Reply:
Done, though instead of changing the name of `combo_coinc_rate` it now simply returns numpy.log(allowed_area) + rateprod